tests/trainer/test_fsdp_checkpoint.py

```diff
@@ -665,6 +665,7 @@ def test_cleanup_sharded_checkpoints(world_size, tmp_path: pathlib.Path, state_d
     run_name = trainer1.state.run_name
     trainer1.fit()
     trainer1.close()
+    # Because os.remove is async
     time.sleep(5)
```

Comment:
```suggestion
    time.sleep(5)  # Because os.remove is async
```
Please detail where it's async and being run? Also I dont think os.remove is async (I lied offline)